pyproject.toml

```
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=0.12"]
```

```suggestion
requires = ["poetry-core>=1.0.7"]
```
Let's align this with antsibull (https://github.com/ansible-community/antsibull/blob/main/pyproject.toml#L2) and antsibull-changelog (https://github.com/ansible-community/antsibull-changelog/blob/main/pyproject.toml#L2).

---

Done. I permitted myself to be lazy and commit via GitHub.